opencga-core/src/main/java/org/opencb/opencga/core/tools/family/PedigreeGraphAnalysisExecutor.java

```diff
@@ -21,6 +21,9 @@
 
 public abstract class PedigreeGraphAnalysisExecutor extends OpenCgaToolExecutor {
 
+    public static final String ID = "pedigree-graph";
```

Comment:
I think that usually ID+Description constants are in the main analysis, rather than in the executor